Go/syntax_test_go.go

```diff
@@ -35,22 +34,30 @@ You may have to disable Go-specific linters when working on this file.
 //                            ^^^^^^^^^^^ comment.line.go
 
     /*
+//  ^^ comment.block.go punctuation.definition.comment.begin.go
 // ^ -comment
```

Comment:
We should probably avoid duplicated tests (of `comment.block). Instead we could ensure correct boundaries by:

```suggestion
// ^ -comment
//  ^^ comment.block.go punctuation.definition.comment.begin.go
//    ^^ comment.block.go - punctuation
```